openshift/frontend.dc.yaml

```diff
@@ -62,6 +62,8 @@ objects:
             value: "${PATH_ROOT}"
           - name: REACT_APP_CHES_ROOT
             value: "${PATH_ROOT}"
+          - name: REACT_APP_CHES_PERFORMANCE
+            value: "${REACT_APP_CHES_PERFORMANCE}"
```

Comment:
The value should be a frontend.dc.yaml parameter that we can populate from the command-line or our Jenkinsfiles.  For example, there is a HOST_URL parameter (line 157), and we pass that in from Jenkinsfile and it is used to populate the OpenShift deployer with the REACT_APP_PUBLIC_URL (line 70).  In this case, we can set a default value for the parameter to our PROD url, and only override when needed (not yet, but the flexibility is there - so in the future we can decide how we want to override).